pkg/kubectl/cmd/config/delete_context_test.go

```diff
@@ -65,7 +65,7 @@ func (test deleteContextTest) run(t *testing.T) {
 	pathOptions.EnvVar = ""
 
 	buf := bytes.NewBuffer([]byte{})
-	cmd := NewCmdConfigDeleteContext(buf, pathOptions)
+	cmd := NewCmdConfigDeleteContext(buf, nil, pathOptions)
```

Comment:
Sounds reasonable tome. Some defensive programming would be better here. Updated